bigbluebutton-html5/private/locales/en.json

```diff
@@ -202,6 +202,22 @@
     "app.audioModal.audioChoiceLabel": "How would you like to join the audio?",
     "app.audioModal.audioChoiceDesc": "Select how to join the audio in this meeting",
     "app.audioModal.closeLabel": "Close",
+    "app.audioModal.yes": "Yes",
+    "app.audioModal.no": "No",
+    "app.audioModal.echoTestTitle": "Speak a few words, Do you hear yourself? Does everything seems correct?",
```

Comment:
How about we copy this string from Flash?

> This is a private echo test.  Speak a few words.  Did you hear audio?

Reply:
sure.